larsim/IonizationScintillation/IonAndScint_module.cc

```diff
@@ -136,6 +160,7 @@ namespace larg4 {
       fISAlg = std::make_unique<ISCalcNESTLAr>(fEngine);
     else
       mf::LogWarning("IonAndScint") << "No ISCalculation set, this can't be good.";
+    if (fGapTool) std::cout << "Tool on, begin job" << std::endl;
```

Comment:
the couts should use the messaging service.  There's an example of how to use the messaging service once line above this.